docs/utils/threading/worker-pool.md

```diff
@@ -148,3 +148,129 @@ functionality in Thunder, which will be further described in a different documen
 ## Conclusions
 
 To sum up, the main idea is not to reinvent the wheel. When creating plugins, developers should keep in mind they are working on a large system in an embedded environment. Because of that from an architectural point of view, a different set of rules applies than when working on developing PC or even mobile applications. We all have to be aware of limitations like the low amount of memory available, the difficulty of keeping a system with dozens of plugins scalable, or even the necessity to use abstractions to achieve portability. These are the main reasons why it is essential to use functionalities that are already given, instead of making things suboptimally on your own.
+
+## Scheduler
+### Overview
+`Workerpool` has the ability to schedule `jobs` using the `Scheduler` class. This can be very helpful when you do not want to start a `job` immediately but at a specific time. To learn more about how to create a `job` read [Workerpool](../threading/worker-pool) section of documentation.
+
+ To schedule a `job` we use several methods for this:
+
+* `Schedule()`
+* `Reschedule()`
+* `Revoke()`
+
+Using these methods, we can schedule the performance of a task, change the date on which it will be performed or cancel it altogether. 
+### Scheduler methods
+
+#### Schedule
+`Schedule()` is used to plan when to perform the `job` we specified. Using this method is very simple. We specify the `job` and the date on which to start performing it in the appropriate format.
```

Comment:
Maybe instead of "We specify the `job` and the date on which to start performing it in the appropriate format", we could change it to something like: "We just need to specify the `job` with a point in time when it should be performed."
I think it sounds more clear, what do you think?